practice-one/src/pages/Details/index.tsx

```diff
@@ -0,0 +1,5 @@
+const DetailsPage = () => {
```

Comment:
folder should be ProductDetail, the same for file name

Reply:
I think should be product-detail for folder name instead